galaxy/main/models.py

```diff
@@ -1060,6 +1060,12 @@ def content_counts(self):
             .annotate(count=models.Count('content_type__name')) \
             .order_by('content_type__name')
 
+    @property
+    def community_survey_count(self):
```

Comment:
This property will produce +1 query for each repository item in list.
Please consider using join + count.